### ISSUE TITLE
Update WCAG mapping for a few rules

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#3703]
Update WCAG mapping for the following rules to match the mappings in axe-core

Rule name | Current WCAG mapping | Expected  WCAG mapping
-- | -- | --
aria-allowed-attr | WCAG 4.1.1, WCAG 4.1.2 | WCAG 4.1.2
aria-required-attr | WCAG 4.1.1, WCAG 4.1.2 | WCAG 4.1.2
aria-roles | WCAG 1.3.1, WCAG 4.1.1, WCAG 4.1.2 | WCAG 4.1.2
aria-valid-attr | WCAG 4.1.1 | WCAG 4.1.2
aria-valid-attr-value | WCAG 4.1.1, WCAG 4.1.2 | WCAG 4.1.2
frame-title | WCAG 2.4.1 | WCAG 2.4.1, WCAG 4.1.2

## Context

- AI-Web Version & Environment: Prod 2.22.0

**Comment by ferBonnin** [comment]
Per conversation with Peter, let's try to tackle this as part of the axe-core update feature, if possible. cc @lisli1

**Comment by ferBonnin** [comment]
@dbjorge can this be marked as resolved?

**Comment by dbjorge** [comment]
Yep! This should be resolved as of prod 2.23.0

**Comment by ferBonnin** [comment]
@iamrafan could you please verify and close?

**Comment by iamrafan** [comment]
Validate in prod 2.23.0
